.github/workflows/main.yml

```diff
@@ -17,6 +18,7 @@ on:
       - test/**
       - bindings/**
       - binding.gyp
+      - .github/workflows/*
```

Comment:
Tbh kinda wonder how worth using these paths are since the jobs are so fast anyway.